# PedestrianPositions.msg
geometry_msgs/Point[] positions